doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+# SSH Slaves plugin
+This plugin allows you to manage agent running on *nix machines over SSH. It adds a new type of agent launch method.
```

Comment:
Need to escape the `*` here to be safe. Even GitHub is confused about the italics in this file.